dNet/dServer.cpp

```diff
@@ -68,7 +70,16 @@ dServer::dServer(const std::string& ip, int port, int instanceID, int maxConnect
 			LOG("%s Server is listening on %s:%i with encryption: %i", StringifiedEnum::ToString(serverType).data(), ip.c_str(), port, int(useEncryption));
 		else
 			LOG("%s Server is listening on %s:%i with encryption: %i, running zone %i / %i", StringifiedEnum::ToString(serverType).data(), ip.c_str(), port, int(useEncryption), zoneID, instanceID);
-	} else { LOG("FAILED TO START SERVER ON IP/PORT: %s:%i", ip.c_str(), port); return; }
+	} else {
+		LOG("FAILED TO START SERVER ON IP/PORT: %s:%i", ip.c_str(), port);
+#ifdef DARKFLAME_PLATFORM_LINUX
+		if (mServerType == ServerType::Auth) {
+			const auto cwd = std::filesystem::current_path();
```

Comment:
```suggestion
			const auto cwd = BinaryPathFinder::GetBinaryDir();
```
This won't give the correct path when being run from outside the binary's current directory